rpcs3/util/sysinfo.cpp

```diff
@@ -138,6 +138,33 @@ bool utils::has_fma4()
 	return g_value;
 }
 
+bool utils::has_erms()
+{
+	static const bool g_value = get_cpuid(0, 0)[0] >= 0x7 && (get_cpuid(7, 0)[1] & 0x200) == 0x200;
+	return g_value;
+}
+
+bool utils::has_fsrm()
+{
+	static const bool g_value = get_cpuid(0, 0)[0] >= 0x7 && (get_cpuid(7, 0)[3] & 0x10) == 0x10;
+	return g_value;
+}
+
+u32 utils::get_rep_movsb_threshold()
+{
+	u32 g_value = 0xFFFFFFFF;
```

Comment:
Some random nitpick: I think the only reason for these to be g_value was that they were static (which should be s_value anyway).
That being said, I think you can easily make this variable static as well (I know it's already assigned to a static later in SPUThread.cpp)

Reply:
It's best to make it static as well.

Reply:
I made it static